.github/workflows/release-tags.yml

```diff
@@ -21,7 +21,17 @@ jobs:
       - name: Fixup URL in nix-installer.sh
         run: |
           sed -i "s@https://install.determinate.systems/nix@https://install.determinate.systems/nix/tag/$GITHUB_REF_NAME@" nix-installer.sh
-      - name: Publish Release (Tag)
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.AWS_S3_UPLOAD_ROLE }}
+          aws-region: us-east-2
+      - name: Publish Release to S3 (Tag)
+        env:
+          AWS_BUCKET: ${{ secrets.AWS_S3_UPLOAD_BUCKET }}
+        run: |
+          ./upload_s3.sh "$GITHUB_SHA" "$GIT_ISH"
```

Comment:
Not quite -- the first argument should be the tag (`$GITHUB_REF_NAME`), and the second should be the git SHA (`$GITHUB_SHA`).

Reply:
Basically, the first argument is the name of the directory that will go into the bucket, while the second is the git commit hash that it will point to (which will also be a directory in the bucket).

Reply:
Agh